compiler/src/iree/compiler/Dialect/HAL/Target/VulkanSPIRV/VulkanSPIRVTarget.cpp

```diff
@@ -162,6 +162,10 @@ class VulkanSPIRVTargetBackend : public TargetBackend {
     buildSPIRVCodegenPassPipeline(passManager, /*enableFastMath=*/false);
   }
 
+  void buildLinkingPassPipeline(OpPassManager &passManager) override {
+    buildSPIRVLinkingPassPipeline(passManager);
+  }
```

Comment:
Just checking - does this target still work if linking is disabled? https://github.com/openxla/iree/blob/69398bc9fa793b8ce7e89bdeb12e196f819a60e0/compiler/src/iree/compiler/Dialect/HAL/Transforms/Passes.cpp#L38-L43 https://github.com/openxla/iree/blob/69398bc9fa793b8ce7e89bdeb12e196f819a60e0/compiler/src/iree/compiler/Dialect/HAL/Transforms/Passes.cpp#L355-L362

Reply:
Yup it should be no problem to turn off the linking, which would mean going to the old one-entry-point and one-shader-module per executable.